### ISSUE TITLE
Publish ruukh

### ISSUE THREAD

**Issue by csharad** [csharad/ruukh#4]
Before publishing a first version, it would be nice to have `web-sys` published to showcase much more complex usage/examples.

Blocked by: rustwasm/wasm-bindgen#613

**Comment by csharad** [comment]
Now [released](https://github.com/csharad/ruukh/releases/tag/0.0.2).
